### PR TITLE
Reader configuration

### PR DESCRIPTION
**Pull request by kama1**
- Ability to change background
- Ability to change font family and size

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,8 +10,8 @@
   <link rel="icon" type="image/x-icon" href="assets/icon/favicon.ico">
   <link rel="manifest" href="manifest.json">
   <meta name="theme-color" content="#4e8ef7">
-
   <script src="assets/epubjs/build/epub.min.js"></script>
+  <script src="assets/epubjs/build/libs/zip.min.js"></script>
```
Comment (janpio) [inline src/index.html]: Unrelated change? This is "only" needed to open .epub files instead of the folders I used for now, correct?

Comment (kama1) [inline src/index.html]: yes, you are right, I added this line if somebody wants to read an epub file instead of extracted folder.
